test_suite/generic/test_generic.py

```diff
@@ -683,7 +717,8 @@ def __get_kdump_config_by_rhel_version(self, rhel_version):
                 ]
             }
             if rhel_version >= 9.1:
-                conf['sysconfig_kdump'][2] = 'KDUMP_COMMANDLINE_REMOVE="hugepages hugepagesz slub_debug quiet log_buf_len swiotlb cma hugetlb_cma ignition.firstboot"'
+                conf['sysconfig_kdump'][
+                    2] = 'KDUMP_COMMANDLINE_REMOVE="hugepages hugepagesz slub_debug quiet log_buf_len swiotlb cma hugetlb_cma ignition.firstboot"'
```

Comment:
Is this format intentional? It seems a bit weird

Reply:
The linter messed it up. I'll fix it.